index.js

```diff
@@ -152,18 +159,19 @@ const preprocessor = (options = {}) => {
       log(`update ${filePath}`)
       // we overwrite the cached bundle promise, so on subsequent invocations
       // it gets the latest bundle
-      const bundlePromise = bundles[filePath] = bundle()
-      .finally(() => {
+      const bundlePromise = (bundles[filePath] = bundle().finally(() => {
```

Comment:
I'd rather see this without the bizarre parentheses. I guess that means it needs to be 2 lines:

```
const bundlePromise = bundle.finally(() => {
// ...
bundles[filePath] = bundlePromise
```